css/style.css

```
@@ -0,0 +1,106 @@
+/***************************/
+/* HEADER */
+/**************************/
+.header {
+  display: flex;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  position: absolute;
+  gap: 2.2rem;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.promo {
+  background-image: url("/img/promo-background.png");
+  box-shadow: inset 0px 140px 40px rgb(0 0 0 / 30%);
+  min-height: 75rem;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  color: #fff;
+  font-size: 3.6rem;
+  height: 12rem;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+  align-self: stretch;
+}
+
+.header-nav-list {
+  display: flex;
+  list-style: none;
+  gap: 0.8rem;
+}
+
+li {
+  align-items: center;
+  display: flex;
+  position: relative;
+}
+
+.header-nav-link:link,
+.header-nav-link:visited {
+  padding-left: 2rem;
+  padding-right: 2rem;
+  text-decoration: none;
+  color: #fff;
+  font-size: 2.1rem;
+  font-family: "Gentium Plus", serif;
+  font-weight: 700;
+}
+
+.header-nav-link:hover::after,
+.header-nav-link:active::after,
+.header-nav-link.is-active::after {
+  opacity: 0.8;
+  transition: all 0.5s;
+}
+
+.header-nav-link::after {
+  display: flex;
+  align-items: flex-start;
+  opacity: 0;
+  content: "";
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  height: 0.3rem;
+  width: 100%;
+  background: #fff;
+}
+
+.header-social {
+  display: flex;
+  list-style: none;
+  gap: 2rem;
+}
+
+.header-social-link:link,
+.header-social-link:visited {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 3.6rem;
+  height: 3.6rem;
```

3.6 seems to big for such element
```suggestion
  width: 3.2rem;
  height: 3.2rem;
```